end_users/README.md

```diff
@@ -22,3 +28,79 @@ For more on these options and what they mean, read the [documentation on merge m
+
+### Bug Report Requirements
+
+* A descriptive title. Use keywords so others can find your bug to avoid duplicates and improve its visibility.
+
+* Limited to one bug. If you have another bug to report, please raise a separate issue.
+
+* Links to your project's:
+  * relevant BonnyCI logs
+  * BonnyCI configuration file, `.bonnyci`
+  * Any other miscellaneous scripts that BonnyCI might be kicking off.
+
+* The expected result or output that BonnyCI will produce.
+
+* The actual result or output that BonnyCI is currently producing.
+
+* The following details about your environment:
+  * BonnyCI version or latest SHA if you are running from source.
+  * Any tools you may be installing that are relevant to the issue (ex: if BonnyCI is running a linter, include a link to this linter).
+  * The primary language(s) of your project.
+
+* The red `bug` label.
+
+* Please do not add a milestone label. This will be added by the BonnyCI Admin team.
+
+The BonnyCI admin team will be alerted as soon as the bug report is summitted, and a team member will address it as soon as possible. Further clarification or details may be required depending on the situation.
+
+### Bug Lifecycle
+
+1. New bug is filed; awaiting review.
+
+2. Triaged in bug review and assigned to a developer.
+
+3. Developer begins working on it -- bug is tagged `fix_in_progress`.
+
+4. Developer opens pull request with a fix, which must be reviewed and tested -- a link to the pull request appears in the bug's activity stream.
+
+5. Pull request is merged, and the bug's filer is pinged to verify that it's fixed -- bug is tagged `fixed_but_not_closed`.
+
+6. Filer agrees that it's fixed -- bug is closed, and its milestone is set to the release the fix landed in.
+
+## Feature Requests
+
+If you wish for a feature to be added to BonnyCI, please submit it to the [projman issue tracker](https://github.com/BonnyCI/projman/issues). Make sure that a similar feature request has not already been filed, and post on that feature request's issue conversation if it has. The feature request will be reviewed and added to a milestone if it is deemed beneficial to the project.
+
+### Feature Request Requirements
+
+* A descriptive title. Use keywords so others can find your feature request to avoid duplicates and improve its visibility.
+
+* Limited to one feature. If you have another feature to request, please raise a separate issue.
+
+* Short and precise requirements such as "BonnyCI support for Zuul v3". If the feature request has multiple moving parts, it may be transferred to an EPIC that is broken down into smaller feature additions.
+
+* A short explanation (2-3 sentences) of why you think the feature will be beneficial to the long-term health of the project.
+
+* The `feature request` label available in the issue tracker.
```

Comment:
Also, the `feature request` label does not exist yet. Should this be [`enhancement`](https://github.com/BonnyCI/projman/labels/enhancement)?